jwst/badpix_selfcal/badpix_selfcal_step.py

```diff
@@ -103,7 +103,15 @@ def process(self, input, selfcal_list=None, bkg_list=None):
         selfcal_list = [input_sci] + selfcal_list
         selfcal_3d = []
         for i, selfcal_model in enumerate(selfcal_list):
-            selfcal_3d.append(selfcal_model.data)
+            # If working with MIRI MRS data, subtract any pedestal residual dark
+            # using the median of count rates from between the channels
+            if (input_sci.meta.instrument.detector.upper() == 'MIRIFUSHORT'):
+                selfcal_3d.append(selfcal_model.data - np.nanmedian(selfcal_model.data[50:1024-50, 503:516]))
+            elif (input_sci.meta.instrument.detector.upper() == 'MIRIFULONG'):
+                selfcal_3d.append(selfcal_model.data - np.nanmedian(selfcal_model.data[50:1024-50, 474:507]))
+            else:
+                selfcal_3d.append(selfcal_model.data)
```

Comment:
Style nit-pick incoming: Oftentimes in the pipeline when we want to hard-code numbers like this, we create an all-caps variable at the top of the file (just below imports) [like this example](https://github.com/spacetelescope/jwst/blob/master/jwst/barshadow/bar_shadow.py).  I think it would be good to do this with the pedestal indices, e.g. `PEDESTAL_IDX_LONG=(50,974,474,507)` and then call that variable where needed.